samcli/commands/build/build_context.py

```diff
@@ -241,6 +242,7 @@ def run(self):
             UnsupportedBuilderLibraryVersionError,
             ContainerBuildNotSupported,
             InvalidBuildGraphException,
+            ContainerStartTimeoutException,
```

Comment:
adding here because it wouldn't be caught and re-raised as a UserException otherwise and it would show the whole stack trace in a not very user friendly way (for building with `--use-container` option since it also goes through the code that waits for the port).